uiowa/uiowa
# Admissions: Refactor Area of Study college field to use taxonomy

Resolves #3223 

# How to test

* `blt ds --site=admissions.uiowa.edu`
* `drush @admissions.local uli admin/structure/taxonomy/manage/colleges/add` and create an entry for Tippie College of Business ([the original, for reference](https://admissions.uiowa.edu/colleges/tippie-college-business)).
* Visit http://admissions.local.drupal.uiowa.edu/node/1021/edit and add "Tippie College of Business" in the "College(s)" field.
* Save the page and observe that the link shows up in the "Related links" section.
* Click the link to the term page. Observe that "Business" shows up underneath the college information.
